cylc/uiserver/app.py

```diff
@@ -126,13 +129,17 @@ class CylcUIServer(ExtensionApp):
 
     name = 'cylc'
     app_name = 'cylc-gui'
-    default_url = "/cylc"
     load_other_extensions = True
     description = '''
-    Cylc - A user interface for monitoring and controlling Cylc workflows.
+    Cylc gui - A user interface for monitoring and controlling Cylc workflows.
     '''  # type: ignore[assignment]
     examples = '''
-        cylc gui    # start the cylc GUI
+        cylc gui                  # Start the Cylc GUI (At the dashboard page)
+cylc gui [workflow]       # Start the Cylc GUI (at the workflow page),
+cylc gui --new [workflow] # Start the Cylc GUI (at the workflow page), with a
```

Comment:
I am seeing it there but not in the list of options. Is there a way to include it?
```
Options
=======
The options below are convenience aliases to configurable class-options,
as listed in the "Equivalent to" description-line of the aliases.
To see all configurable class-options for some <cmd>, use:
    <cmd> --help-all

--debug
    set log level to logging.DEBUG (maximize logging output)
    Equivalent to: [--Application.log_level=10]
--show-config
    Show the application's configuration (human-readable format)
    Equivalent to: [--Application.show_config=True]
--show-config-json
    Show the application's configuration (json format)
    Equivalent to: [--Application.show_config_json=True]
--generate-config
    generate default config file
    Equivalent to: [--JupyterApp.generate_config=True]
-y
    Answer yes to any questions instead of prompting.
    Equivalent to: [--JupyterApp.answer_yes=True]
--log-level=<Enum>
    Set the log level by value or name.
    Choices: any of [0, 10, 20, 30, 40, 50, 'DEBUG', 'INFO', 'WARN', 'ERROR', 'CRITICAL']
    Default: 30
    Equivalent to: [--Application.log_level]
--config=<Unicode>
    Full path of a config file.
    Default: ''
    Equivalent to: [--JupyterApp.config_file]

```

Reply:
No, the options are populated by the definition of our config as traitlets. We cannot add --new as a traitlet option as the config is loaded after the --new would be required.